buildspec/linuxTests.yml

```diff
@@ -33,7 +33,7 @@ phases:
 
       - chmod +x gradlew
       - su codebuild-user -c "./gradlew -PideProfileName=$ALTERNATIVE_IDE_PROFILE_NAME check coverageReport --info --stacktrace --console plain --continue"
```

Comment:
I wonder if we might need to modify this line too, but if we are deleteing this file for the separate buildspecs later, maybe not